libs/pages/settings/src/lib/ui/page-organization-helm-repositories/crud-modal/crud-modal.tsx

```diff
@@ -162,8 +162,130 @@ export function CrudModal({ isEdit, onSubmit, onClose, loading, availableHelmRep
           />
         </>
       )}
+      {watch('kind') === 'OCI_SCALEWAY_CR' && (
```

Comment:
I think you need to reset `config` fields when you change the kind otherwise you'll send unnecessary values

https://github.com/Qovery/console/blob/c340929fde933bdeff4f19bd191e81e004e8e2fe/libs/pages/settings/src/lib/ui/page-organization-container-registries/crud-modal/crud-modal.tsx#L124